### ISSUE TITLE
Lots of duplicate questions from new users

### ISSUE THREAD

**Issue by trainman419** [ros-infrastructure/answers.ros.org#107]
When moderating, I see a lot of duplicate questions from new users.

This suggests that we have a user experience problem with the site - for some reason, new users don't understand that their question has been submitted, and they submit it a second time.

We saw this even before we turned the moderation on, so I suspect it's related to the question submission UI, and not the moderation features.

**Comment by 130s** [comment]
I just noticed there's a `merge` option. Somewhat helpful.

**Comment by gavanderhoorn** [comment]
Bumping this, as it's still an issue. I remove multiple duplicate questions per day.

**Comment by gavanderhoorn** [comment]
And another bump: this is getting really annoying. I daily close and delete something like 10 duplicates now.

Apparently new members start writing a new question without registering, which after they click "Login/Signup to Post" then takes them to the registration screen. After that is finished, they can continue to actually post the question, but apparently that leads to the question being doubleposted (or even tripleposted).

Moderation doesn't help, but as @trainman419 already mentioned, it might be that the submission/registration process has a hickup somewhere.

**Comment by gavanderhoorn** [comment]
Repeating myself a little here, but this is still rather an issue. First thing I do almost daily is close & delete a nr of duplicates, typically posted by new users (but not exclusively).

**Comment by gavanderhoorn** [comment]
This is starting to get old (so this is my last comment here), but the duplicates situation is getting ridiculous. I deleted 8 again today.

It's been a while since I've posted a question to ROS Answers that had to go through moderation, but can we get some (static) overlay for new users that have not yet been through the moderation queue? The overlay could tell them that they're not seeing their question because it's currently being held up in the moderation queue.

I seem to remember Gazebo's AskBot instance does that, but the notice is tiny. Please make it bold font, 32 pts or something.

**Comment by gavanderhoorn** [comment]
I know I promised to stop commenting here in my previous comment, but the nr of duplicates is reaching new levels again.

I'm also getting kinda disappointed here as well: what is the point in reporting issues here if they are never acted upon? Seeing as the "you're question is being held for moderation"-notice is already part of askbot, updating it to make it more noticable should not be too much work.

Does @evgenyfadeev still provide support for the OR instance of AskBot?

**Comment by 130s** [comment]
On a related note, I just realized that I've been contributing to have duplicates passing, by approving new accounts and all the associated posts, instead of cherry-picking each post made by those accounts. Perhaps moderators selectively trim duplicates on the Moderator's view before approving accounts and posts makes things much cleaner practically.

**Comment by gavanderhoorn** [comment]
That seems a bit like a work-around: posting duplicates should just not happen this often. Whether or not the moderation interface has UI that makes dealing with duplicates easier is orthogonal imho.

**Comment by evgenyfadeev** [comment]
@gavanderhoorn did you notice new users posting the same content repeatedly? If this happens due to some confusion, something can be done in the UI to improve in this area.

Perhaps we could force new users to search before posting - since your site already has a lot of content it should be OK, what do you guys think?

**Comment by gavanderhoorn** [comment]
@evgenyfadeev wrote:
> did you notice new users posting the same content repeatedly?

yes. I have had to delete 5 questions today which were duplicates.

> If this happens due to some confusion, something can be done in the UI to improve in this area.

I guess the confusion comes from the fact that new users don't notice that their questions are still held-up in the moderation queue. See https://github.com/ros-infrastructure/answers.ros.org/issues/107#issuecomment-376599861.

**Comment by evgenyfadeev** [comment]
related issues: https://github.com/ros-infrastructure/answers.ros.org/issues/158 https://github.com/ros-infrastructure/answers.ros.org/issues/155

**Comment by gavanderhoorn** [comment]
@evgenyfadeev: any updates?

**Comment by gavanderhoorn** [comment]
And another datapoint: closed about 20 duplicate posts this week alone again.

@evgenyfadeev: is there any chance this will get fixed?

Or are we giving up on AskBot/ROS Answers?

**Comment by evgenyfadeev** [comment]
Hi @gavanderhoorn, there will be updates soon, although not related to this issue.

Not sure how to stop people to from asking duplicate questions, unless they are exact duplicates of previously asked questions or there is a reliable detection of approximate duplicates.

**Comment by gavanderhoorn** [comment]
It's not people posting duplicate questions (as in: different users).

It's duplicates of posts by the same user.

So 3 times the same question, while the user apparently/reportedly has only posted it once.

This is especially the case with new users, but also happens with users that have already been through the moderation queue.

See also the OP by @trainman419:

> When moderating, I see a lot of duplicate questions from new users.

**Comment by gavanderhoorn** [comment]
Example:

 - [344875](https://answers.ros.org/question/344875)
 - [344876](https://answers.ros.org/question/344876)
 - [344877](https://answers.ros.org/question/344877)

All posted by the same user. The first seconds apart, the last a ~1.5 minutes later.

Content is identical.

Could this be a job queue problem?

**Comment by tfoote** [comment]
I think there's a UI issue when people try to post as unauthenticated, and then are prompted to log in. It doesn't get them back to their post which was actually posted. 

> The forum didn't give any indication that the post was created after logging in and instead brought me to a change password page which was odd. Thanks for closing this one out, sorry about the dup.

From the now deleted duplicate: https://answers.ros.org/question/348826/ros2-cli-arguments-distinguishing-from-ros-arguments/

**Comment by gavanderhoorn** [comment]
Yeah. I've closed so many duplicates since February I've lost count.

In extreme cases there are sometimes up to 5 duplicates.

I've stopped posting here as it doesn't seem like this is the place where these issues get attention. Perhaps the main issue tracker for AskBot would be better?

**Comment by evgenyfadeev** [comment]
@gavanderhoorn this is a good place to post, will attend.
